src/views/Home/HomeComponents/SearchbarComponent.vue

```diff
@@ -1,41 +1,32 @@
 <script setup lang="ts">
-import { usePlaceStore } from '@/stores/PlaceStore';
-import { ref, watch, onMounted, onUnmounted } from 'vue';
+import { ref, onMounted, onUnmounted } from 'vue';
 import apiService, { type IPlaceMarker } from '@/services/api.service';
-import { useToast } from 'vue-toastification';
+import { POSITION, useToast } from 'vue-toastification';
 import { useLoadingStore } from '@/stores/LoadingStore';
 import { useSearchStore } from '@/stores/SearchStore';
 import type { IPlaceType } from '@/utils/interfaces/Place';
 
-const place = usePlaceStore();
 const toast = useToast();
 const load = useLoadingStore();
 const search = useSearchStore();
 
 const searchQuery = ref('');
 const searchCategory = ref<IPlaceType>('onsen');
-const errorMessage = ref('');
 const emit = defineEmits(['search']);
 
-// Watch the searchQuery so any changes clear the error message
-watch(searchQuery, () => {
-  if (errorMessage.value) {
-    errorMessage.value = '';
-  }
-});
-
 const performSearch = async () => {
   // Validate the search term
-  if (searchQuery.value.length < 5) {
-    errorMessage.value = 'Search term is not long enough.';
+  if (searchQuery.value.length < 4) {
+    toast.error("Search term is not long enough", {
+      timeout: 2500,
+      position: POSITION.TOP_LEFT
+    })
```

Comment:
Good idea! The feedback last week suggested we might have place names with 2 letters, but I think < 4 is enough.